features/in_onderzoek.feature

```diff
@@ -166,188 +165,283 @@ Functionaliteit: in onderzoek
 
     Abstract Scenario: persoon heeft <veld> in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft de volgende 'persoonInOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      | naam                                        | waarde                |
+      | burgerservicenummer                         | 555550001             |
+      | aanduiding gegevens in onderzoek (01.83.10) | <in onderzoek waarde> |
+      | datum ingang onderzoek (01.83.20)           | 20220307              |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | <veld>                          |
-      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'inOnderzoek' gegevens
-      | naam                                    | waarde       |
-      | <veld>                                  | true         |
-      | datumIngangOnderzoekPersoon.datum       | 2022-03-07   |
-      | datumIngangOnderzoekPersoon.type        | Datum        |
-      | datumIngangOnderzoekPersoon.langFormaat | 7 maart 2022 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                                                | waarde       |
+      | <veld>                                              | <waarde>     |
+      | inOnderzoek.<veld>                                  | true         |
+      | inOnderzoek.datumIngangOnderzoekPersoon.datum       | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoekPersoon.type        | Datum        |
+      | inOnderzoek.datumIngangOnderzoekPersoon.langFormaat | 7 maart 2022 |
 
       Voorbeelden:
-      | waarde | veld                |
-      | 010120 | burgerservicenummer |
-      | 010410 | geslacht            |
+      | in onderzoek waarde | veld                | waarde    |
+      | 010120              | burgerservicenummer | 555550001 |
+      | 010410              | geslacht            |           |
```

Comment:
Als ik dit zo lees dan wordt in het geval van regel 188 in de response een leeg 'geslacht' property opgenomen. Is dat inderdaad wat je zou verwachten?

Reply:
Wat bedoel jij met leeg geslacht property?
```
{
  'geslacht': ''
}
```
of
```
{
  'geslacht': null
}
```
of geen geslacht
```
{
}
```
of weet je nog een andere optie?
In de HC APIs worden optie 2 en optie 3 als hetzelfde gezien.
Bij regel 188 krijgt je optie 3

Reply:
Regel 179 lijkt mij te duiden op de opties 1 of 2, er wordt immers expliciet aangegeven dat de response het veld 'geslacht bevat.

Reply:
#1202